features/pagination.feature

```diff
@@ -10,6 +10,43 @@ Feature: Use Pagination to filter result
     | 2  | Banana                | fruit,yellow | 2001-07-21 | 2003-08-30   | true      | 0.50$ |
     | 3  | Sausages              | meat         | 2010-04-11 | 2005-02-01   | true      | 5.30$ |
 
-  Scenario: Return all the data set when no parameters are given
-    When I request the url "/dataset"
+  Scenario: Return all the data when no parameters are given
+    When I request the url "/resource" with the page parameters
+    | |
+    Then I should have the following ids "1,2,3"
+
+  Scenario: Return all the data based on start and limit array format
+    When I request the url "/resource" with the page parameters
+      | start | limit |
+      | 1     | 1     |
```

Comment:
should limit be -1 or null to denote a none limit?

Reply:
the limit would be unset when none limit ie. `/resource` or `/resource?start=3`

Reply:
?page[start]=3

and the page[limit] could be optional